packages/cli/src/zosjobs/submit/Submit.definition.ts

```diff
@@ -27,6 +28,7 @@ export const SubmitDefinition: ICommandDefinition = {
     description: strings.DESCRIPTION,
     children: [
         DataSetDefinition,
+        USSFileDefinition,
         LocalFileDefinition,
         StdinDefinition
     ]
```

Comment:
Should these be listed in alphabetical order? A minor detail, but I noticed in one of the snapshots the order corresponds to the order they're defined here:
```
Available commands are \\"data-set, uss-file, local-file, stdin\\".
```